Makefile

```diff
@@ -2,7 +2,7 @@ VERSION := $(shell git describe --tags)
 DIST_DIRS := find * -type d -exec
 
 build:
-	go build -o glide -ldflags "-X main.version=${VERSION}" glide.go
+	go build -o glide -ldflags "-X main.version=$(VERSION)" glide.go
```

Comment:
I'm not sure I understand the change here. `$(VERSION)` tells the shell script to execute `VERSION` and return the result of that. `${VERSION}` returns the value of the variable. In this case they will have the same result. But, I'm not sure of the motivation for the change.

Can you explain? Is there something about shell scripting I'm not aware of (which could easily be the case because I rarely do it)?

As the one who runs the builds, the current implementation generates the correct version. Are you seeing something different?